### DISCUSSION TITLE
Amazon Echo in Home Assistant con doppia integrazione?

### DISCUSSION THREAD

**Discussion by Topolynx** [indomus/forum#2301] [HUB: Home Assistant]
Ciao a tutti! Tempo fa avevo seguito la guida inDomus per integrare il mio Echo show in HA con l’integrazione Alexa Media Player. Oggi vorrei però esporre ad Echo le entità di Home Assistant utilizzando Haaska come da altra guida inDomus, e volevo sapere da qualcuno dì voi che lo ha già fatto se devo prima rimuovere l’altra integrazione per evitare di generare conflitti tra le due. Sicuramente vorrei aggiornare la componente domotica di Alexa e disfarmi delle skills Tuya, Meross e Netatmo. Qualche consiglio? Grazie tante!

**Comment by indomus** [comment]
> Ciao a tutti! Tempo fa avevo seguito la guida inDomus per integrare il mio Echo show in HA con l’integrazione Alexa Media Player. Oggi vorrei però esporre ad Echo le entità di Home Assistant utilizzando Haaska come da altra guida inDomus, e volevo sapere da qualcuno dì voi che lo ha già fatto se devo prima rimuovere l’altra integrazione per evitare di generare conflitti tra le due. Sicuramente vorrei aggiornare la componente domotica di Alexa e disfarmi delle skills Tuya, Meross e Netatmo. Qualche consiglio? Grazie tante!

Sono due cose totalmente indipendenti l'una dall'altra. Procedi tranquillamente

**Comment by Topolynx** [comment]
Grazie!
